Makefile documentation

chaitu6022 opened the pull request:
What this PR does?

- This PR contains the documentation for the Make files and scripts 
- Removed commented code for cleanup.

Issues numbers
#13 
#12

chaitu6022 committed:
Node Port services files and updated Make command scripts

chaitu6022 committed:
Makefile documentation

chaitu6022 committed:
Update README.md

chaitu6022 committed:
Update README.md

chaitu6022 committed:
GCP compute engine quotas documentation

bradfordcp commented on gcp/README.md:
````diff
@@ -104,6 +101,9 @@ Execute the following commands on the linux machine in order to setup gcloud cli
 ```
+### GCP Quotas
+
+If you created your Google cloud account newly, Google Compute Engine enforces quotas on resource usage for a variety of reasons. For example, quotas protect the community of Google Cloud users by preventing unforeseen spikes in usage, Google keep some soft limitations on the resources, you can always make a request to increase your quota limit. If you are planning to deploy k8ssandra cluster on  ...
````
With a running installation can we check what this value should be _per k8ssandra deployment_ and provide a minimum here for a single deployment instead of a static `50`?

bradfordcp replied:
It looks like this is captured in https://github.com/k8ssandra/k8ssandra-terraform/issues/11.

Resolving.

chaitu6022 committed:
GCP Makefile documentation

What this commit does?
- Removing make validate stage from the make commands list.
- Validate stage attached in init.sh
- Added documentation for the make commands
- Updated scripts to use variable to choose a cloud provider.

chaitu6022 committed:
Shorten IAM resource name's

chaitu6022 committed:
Resource naming conventions documentation

chaitu6022 committed:
Update README.md

chrislovecnm commented:
@bradfordcp can you file an issue? I am going to merge this
